widget/entry.go

```diff
@@ -1033,6 +1054,13 @@ func (e *Entry) selectingKeyHandler(key *fyne.KeyEvent) bool {
 	case fyne.KeyBackspace, fyne.KeyDelete:
 		// clears the selection -- return handled
 		e.setFieldsAndRefresh(e.eraseSelection)
+		e.propertyLock.Lock()
+		content := e.textProvider().String()
+		if e.OnChanged != nil {
+			e.OnChanged(content)
```

Comment:
Same comment as above applies here.

```suggestion
		if e.OnChanged != nil {
			e.OnChanged(e.Text)
```